pkg/cluster/config/default.go

```diff
@@ -28,21 +28,9 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-// SetDefaultsConfig sets uninitialized fields to their default value.
-func SetDefaultsConfig(obj *Config) {
+// SetDefaults_Config sets uninitialized fields to their default value.
+func SetDefaults_Config(obj *Config) {
```

Comment:
Typically defaults are applied to *external* API versions before converting to the internal representation. This allows for default values to be versioned as well as fields etc.

Applying defaults to the internal representation will mean all API versions will always have to have the same default values.

Reply:
Now defaults are on the external API